mpt/core/pom.xml

```diff
@@ -85,9 +85,9 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
+            <!-- <plugin>
                 <groupId>org.apache.felix</groupId>
-                <artifactId>maven-bundle-plugin</artifactId>
+                <artifactId>maven-bundle-plugin </artifactId>
```

Comment:
+1 to turn this plugin off

Reply:
hmm actually we can't at least not easily 
- the output of this plugin is used as in input in the jar plugin see line 67
- this plugin is not responsible for the forks :D I was going to revert that change

Reply:
I POCed dropping the plugin and letting jar-plugin create the manifest 
the diff in the manifest file is quite heavy and I have no idea what the consequences could be
![image](https://github.com/apache/james-project/assets/22979/9857a441-e8d1-4e2f-ba1c-3611f7a48bcf)
some trivial properties can be replicated or ignored but no so sure about Import-Package